learning_cp/knapsack/Project.toml

```diff
@@ -1,6 +1,7 @@
 [deps]
```

Comment:
Some deps seem unneeded. I think this should be double checked. Shouldn't we add SeaPearlExtras to standardize our experiments?